### PR TITLE
Add CentOS7&8 and AmazonLinux2 support to Cloudformation template.

### PR DESCRIPTION
**Pull request by Scottpedia**
All new OS(s) are fully tested.

### CODE REVIEW

**Comment by Scottpedia** [comment]
@hwdsl2 By the way, happy new year!

File: aws/cloudformation-template-ipsec [hunk 1]
```diff
@@ -832,4 +884,4 @@
             "Value": "https://github.com/hwdsl2/setup-ipsec-vpn#next-steps"
         }
     }
-}
+}
```
Comment (hwdsl2) [inline aws/cloudformation-template-ipsec:887]: It is recommended to leave a newline at the end of the file (looks like it was removed).

**Review by hwdsl2 — CHANGES_REQUESTED** [review]
@Scottpedia Thank you for contributing! And happy new year to you too. I've made some minor suggestions in this PR. Also the README file may need updating, to add CentOS 7/8 and Amazon Linux 2, as well as to list the default usernames under section "How to connect to the server via SSH after deployment?" I can update the README files if you prefer.

**Comment by Scottpedia** [comment]
@hwdsl2  I have made the requested changes. Please take a look!

**Comment by Scottpedia** [comment]
Maybe you can come back tomorrow so that I can make the necessary changes to the documents tonight.

**Comment by hwdsl2** [comment]
@Scottpedia Thank you for the changes. The "yum check-update" parts isn't necessary because `yum` will update the package index before installing packages. Also it looks like you added a blank line before the last line of the file, on line 887. It doesn't affect the functionality though. Please take your time with the doc updates.

**Comment by Scottpedia** [comment]
@hwdsl2  No problem. Nice working with you. Have a good night!

**Comment by Scottpedia** [comment]
@hwdsl2  I have updated the docs in English. Let me know if any further changes need to be made. By the way, could please you update the Chinese version of it? I found myself not very familiar with the technical terms used here in Chinese.

**Pull request merged**